cosim/include/common/bsg_axil.h

```diff
@@ -119,7 +119,7 @@ template <unsigned int A, unsigned int D> class axilm {
   // Wait for (low true) reset to be asserted by the testbench
   void reset(void (*tick)()) {
     printf("bp_zynq_pl: Entering reset\n");
-    while (this->p_aresetn == 1) {
```

Comment:
These polarities shouldn't invert

Reply:
Aren't those signals low true signals?

Reply:
Oh, yeah I think you're right. So that was just wrong before?

Reply:
Basically they were just like normal high true signals which is not true.

Reply:
Kk